src/routes/+page.server.ts

```diff
@@ -29,31 +30,38 @@ const query = gql`
             videoSubhead
           }
         }
+        popularResourcesListCollection(limit: 9) {
+          items {
+            pageMetadata {
+              sys {
+                id
+              }
+            }
+            title
+            subheading
+          }
+        }
         featuredServiceCardsCollection {
           items {
-            ... on ServiceGroup {
-              pageMetadata {
-                ... on PageMetadata {
+            pageMetadata {
+              sys {
+                id
+              }
```

Comment:
I thought that `... on ServiceGroup`, `... on PageMetadata`, and other inline fragments were required whenever we were using a Contentful reference. Is that not the case?

Reply:
I'm pretty sure that's only necessary if there's more than one type that you're querying on, like there is in a heterogenous list. This works and goes thru `npm run check` so I'm in favor of keeping it.